### ISSUE TITLE
Python import errors with new modules

### ISSUE THREAD

**Issue by KrisThielemans** [CCPPETMR/SIRF#164]
we're getting
```
  File "/home/travis/build/CCPPETMR/SIRF-SuperBuild/sources/SIRF/src/xSTIR/pSTIR/tests/test_all.py", line 16, in <module>
    from pUtilities import __licence__, RE_PYEXT
nose.proxy.ImportError: cannot import name '__licence__'
```
see e.g. https://travis-ci.org/CCPPETMR/SIRF/jobs/374451931

**Comment by casperdcl** [comment]
I had fixed this. why has the commit disappeared? grrr

**Comment by KrisThielemans** [comment]
>  why has the commit disappeared?
because you type too fast!

**Comment by casperdcl** [comment]
turns out I fixed it on a different computer and didn't push...

**Comment by KrisThielemans** [comment]
ah. ok then. not the typing. fine. I believe you.

**Comment by casperdcl** [comment]
closed by a27aee8
